pydantic_ai_slim/pydantic_ai/agent.py

```diff
@@ -191,6 +191,7 @@ async def run(
         model: models.Model | models.KnownModelName | None = None,
         deps: AgentDeps = None,
         model_settings: ModelSettings | None = None,
+        execution_limit_settings: ExecutionLimitSettings | None = None,
```

Comment:
```suggestion
        execution_limits: ExecutionLimits | None = None,
```
this would be my preference